tests/functional/tests/security/test_management_fuzzy.py

```diff
@@ -466,3 +467,213 @@ def test_neg_set_nhit_promotion_policy_param_threshold(pyocf_ctx, cm, cls):
+            cache.configure_partition(
+                part_id=1,
+                name="unclassified",
+                min_size=0,
+                max_size=i,
+                priority=0,
+                cache_mode=CACHE_MODE_NONE,
+            )
+            print("\n" + i)
+
+
+@pytest.mark.parametrize("cm", CacheMode)
+@pytest.mark.parametrize("cls", CacheLineSize)
+@pytest.mark.security
+def test_neg_set_ioclass_priority(pyocf_ctx, cm, cls):
+    """
+    Test whether it is possible to add ioclass with invaild priority
+    :param pyocf_ctx: basic pyocf context fixture
+    :param cm: cache mode we start with
+    :param cls: cache line size we start with
+    :return:
+    """
+    # Start cache device
+    cache_device = Volume(S.from_MiB(30))
+    cache = Cache.start_on_device(
+        cache_device,
+        cache_mode=cm,
+        cache_line_size=cls,
+    )
+
+    # Set invalid priority and check if failed
+    for i in RandomGenerator(DefaultRanges.INT16):
+        if i in ConfValidValues.ioclass_priority_range:
+            continue
+        with pytest.raises(OcfError, match="Error adding partition to cache"):
+            cache.configure_partition(
+                part_id=1,
+                name="unclassified",
+                min_size=0,
+                max_size=100,
+                priority=i,
+                cache_mode=CACHE_MODE_NONE,
+            )
+            print("\n" + i)
+
+
+@pytest.mark.parametrize("cm", CacheMode)
+@pytest.mark.parametrize("cls", CacheLineSize)
+@pytest.mark.security
+def test_neg_set_ioclass_cache_mode(pyocf_ctx, cm, cls):
+    """
+    Test whether it is possible to add ioclass with invaild cache mode
+    :param pyocf_ctx: basic pyocf context fixture
+    :param cm: cache mode we start with
+    :param cls: cache line size we start with
+    :return:
+    """
+    # Start cache device
+    cache_device = Volume(S.from_MiB(30))
+    cache = Cache.start_on_device(
+        cache_device,
+        cache_mode=cm,
+        cache_line_size=cls,
+    )
+
+    # Set invalid cache mode and check if failed
+    for i in RandomGenerator(DefaultRanges.INT):
+        if i in CacheMode + [CACHE_MODE_NONE]:
+            continue
+        with pytest.raises(OcfError, match="Error adding partition to cache"):
+            cache.configure_partition(
+                part_id=1,
+                name="unclassified",
+                min_size=0,
+                max_size=100,
+                priority=1,
+                cache_mode=i,
+            )
+            print("\n" + i)
+
+
+@pytest.mark.security
+def test_neg_set_ioclass_name(pyocf_ctx):
+    """
+    Test whether it is possible to add ioclass with invaild name
+    :param pyocf_ctx: basic pyocf context fixture
+    :param cm: cache mode we start with
+    :param cls: cache line size we start with
+    :return:w
+    """
+    invalid_chars_codes = list(range(0, ord(" "))) + list(range(ord("~")+1, 255))
```

Comment:
I think you mean non "printable" characters - there is a list for that string class